humility-arch-cortex/src/debug.rs

```diff
@@ -135,31 +135,257 @@ register!(CPUID, 0xe000_ed00,
     pub revision, _: 3, 0;
 );
 
+register!(SFSR, 0xe000_ede4,
+    #[derive(Copy, Clone)]
+    pub struct SFSR(u32);
+    pub lserr, _: 7;
+    pub sfarvalid, _: 6;
+    pub lsperr, _: 5;
+    pub invtran, _: 4;
+    pub auviol, _: 3;
+    pub inver, _: 2;
+    pub invis, _: 1;
+    pub invep, _: 0;
+);
+
+impl std::fmt::Debug for SFSR {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.lserr() {
+            write!(f, "Lazy state error")?;
+        }
+        if self.lsperr() {
+            write!(f, "Error preserving floting point state")?;
```

Comment:
If more than one bit is set here, the sentences will concatenate without intervening whitespace.

Reply:
Added a space to the end of sentences